modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,46 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.3
+
+This release of {SMProductName} adds developer preview support for Openshift Service Mesh images for ARM based clusters, adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.10 and later versions.
```

Comment:
I thought "S" is capitalized everytime OpenShift is written, including in product names like OSSM ???

```suggestion
This release of {SMProductName} adds developer preview support for OpenShift Service Mesh images for ARM based clusters, adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.10 and later versions.
```